_utility-aliases.scss

```diff
@@ -55,77 +55,85 @@ $text-color--positive: $color--fa-green-text;
 
 /* Color map */
 $color-values: (
-  alt:           $color--alt,
-  base:          $color--base,
-  fa-amber:      $color--fa-amber,
-  fa-blue:       $color--fa-blue,
-  fa-blue-dark:  $color--fa-blue-dark,
-  fa-blue-light: $color--fa-blue-light,
-  fa-cyan:       $color--fa-cyan,
-  fa-green:      $color--fa-green,
-  fa-green-dark: $color--fa-green-dark,
-  fa-orange:     $color--fa-orange,
-  fa-pink:       $color--fa-pink,
-  fa-purple:     $color--fa-purple,
-  fa-red:        $color--fa-red,
-  fa-red-orange: $color--fa-red-orange,
-  fa-yellow:     $color--fa-yellow,
-  gray-1:        $color--gray-1,
-  gray-2:        $color--gray-2,
-  gray-3:        $color--gray-3,
-  gray-4:        $color--gray-4,
-  gray-5:        $color--gray-5,
-  gray-6:        $color--gray-6,
-  gray-7:        $color--gray-7,
-  gray-8:        $color--gray-8,
-  gray-9:        $color--gray-9,
-  gray-10:       $color--gray-10,
-  gray-11:       $color--gray-11,
-  gray-12:       $color--gray-12,
-  gray-13:       $color--gray-13,
-  gray-14:       $color--gray-14,
-  gray-15:       $color--gray-15,
-  gray-16:       $color--gray-16
+  alt:                    $color--alt,
+  base:                   $color--base,
+  fa-amber:               $color--fa-amber,
+  fa-blue:                $color--fa-blue,
+  fa-blue--shade-1:       $color--fa-blue--shade-1,
+  fa-blue--shade-2:       $color--fa-blue--shade-2,
+  fa-blue--shade-3:       $color--fa-blue--shade-3,
```

Comment:
Does a scale like `--shade-1`, `--shade-2`, `--shade-3` actually offer too much flexibility? We want to create a _finite_ scale, but here we could theoretically end with `--shade-17`. It could be argued that the scale of `gray` colours suffer from this same problem, but I see the scale of `gray-[x]` colours as an exception rather than the rule. In practice we've actually needed this many greys; if anything we could do with perhaps 1 or 2 more.

But given how we use (and want to use) the rest of the colour palette in UI work, should we should be enforcing a smaller scale? Using a pattern similar to the existing [`xx-small` > `xx-large`](https://github.com/fac/fa-css-utilities/blob/master/_utility-aliases.scss#L183) would provide 3 lighter variants and 3 darker variants of a colour, in addition to the base colour itself. If 7 variants of a colour aren't enough, perhaps it's a signal that you're actually looking for a different colour altogether, or that there are simply too many colour variants in our system. `fa-amber` is a good example of a colour that may simply have been a darker variant of `fa-yellow`, but exists as it's own colour.